README.md

```diff
@@ -151,6 +151,15 @@ httpd = require('http.server').new(host, port[, { options } ])
 * `idle_timeout` - maximum amount of time an idle (keep-alive) connection will
   remain idle before closing. When the idle timeout is exceeded, HTTP server
   closes the keepalive connection. Default value: 0 seconds (disabled).
+* `socket_timeout` - host resolving timeout in seconds (default 60).
+* TLS options:
+    * `use_tls` is a boolean param to enable tls with tls_options provied below (`false` by default);
```

Comment:
Why do we need this option? As I understand it, for a server should always be installed `ssl_cert_file` and `ssl_key_file`. So we could just enable TLS if any `ssl_*` param is set.

Reply:
Deleted this option and used approach with enabling TLS, if at least on of `ssl_*` option provided.